### PR TITLE
Muon Analysis results tab docs

### LINKED ISSUE #25767: Documentation for muon results table tab
Docs for the new muon results table tab

### PR DESCRIPTION
**Pull request by Harrietbrown**
**Description of work.**
Added documentation for results tab for muon analysis 2.
added sections for data selection window and fit function browser

**To test:**
Build documentation and check that documentations for Muon Analysis 2, its fitting tab and its results table tab are legible, suitably descriptive and that images load propperly.

Fixes #25767.
To be merged after pr #25805

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AnthonyLim23** [comment]
@Harrietbrown  you have some conflicts, a local git merge will probably fix them

**Comment by AnthonyLim23** [comment]
@Harrietbrown could you sort out the merge conflicts?

File: docs/source/interfaces/muon_fitting_tab.rst [hunk 1]
```diff
@@ -19,13 +19,45 @@ This tab is for performing fits on the data.
...
+**Show Selected** If this option is checked the selection box will be limited to only the data sets that have already
+been selected to be output.
+
+**Selection Box** data sets are listed in this section, they can be selected individually by checking the boxes on the
+right side of the box.
+
+**Unselect/Select All** When clicked these buttons will automatically unselect or select all log values currently in the
+selection box
+
+**Run selection** Filter desplayed data sets by run number
```
Comment (AnthonyLim23) [inline docs/source/interfaces/muon_fitting_tab.rst]:
```suggestion
**Run selection** Filter displayed data sets by run number
```

File: docs/source/interfaces/muon_fitting_tab.rst [hunk 2]
```diff
@@ -19,13 +19,45 @@ This tab is for performing fits on the data.
...
 Fit function browser
 ^^^^^^^^^^^^^^^^^^^^
 
+.. image::  ../images/muon_interface_tab_fitting_browser.png
+   :height: 250px
+
 Right clicking in the central box will bring up a context menu for adding or removing functions.
-Once functions or backgrounds are added to the browser you can use to modify the initial guess for a varabile and set it
-If the parameter is to be the same for all of the data sets, the global check box should be ticked.
+
```
Comment (AnthonyLim23) [inline docs/source/interfaces/muon_fitting_tab.rst]: can we link to https://docs.mantidproject.org/v3.13.0/interfaces/Multi-dataset%20Fitting.html (preferably the function browse section)

File: docs/source/interfaces/muon_results_tab.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+.. image::  ../images/muon_interface_tab_results.png
+   :align: right
+   :height: 500px
+
+The results table tab outputs the results from the fitting tab as well as the log values from selected runs into a
+tableworkspace.
+
+Log Values Selection
+^^^^^^^^^^^^^^^^^^^^
+The log values selection area lists log values from the fitted run(s) that can be selected to be output into the results
```
Comment (AnthonyLim23) [inline docs/source/interfaces/muon_results_tab.rst]:
```suggestion
Lists log values from the fitted run(s) that can be selected to be output into the results
```

File: docs/source/interfaces/muon_results_tab.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+**Unselect/Select All** When clicked these buttons will automatically unselect or select all log values currently in the
+selection box
+
+Fit Selection
+^^^^^^^^^^^^^
+The Fit selection area lists outputs from the fitting tab filtered by fitting function, selected functions can be
+outputted to the results table
+
+**Fit function selection** This drop down menu filters the selection box by fit function. Only fits with the same
+function can be output together
```
Comment (AnthonyLim23) [inline docs/source/interfaces/muon_results_tab.rst]:
```suggestion
function can be output together.
```

File: docs/source/interfaces/muon_results_tab.rst [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+**Selection Box** fitting functions are listed in this section, they can be selected individually by checking the boxes
+on the right side of the box.
+
+**Unselect/Select All** When clicked these buttons will automatically unselect or select all functions currently in the
+selection box
+
+Output
+^^^^^^
+
+**Table Name** The selected results will be output to a tableworkspace with the name given here.
```
Comment (AnthonyLim23) [inline docs/source/interfaces/muon_results_tab.rst]:
```suggestion
**Table Name** The output tableworkspace's name is given here.
```

File: docs/source/interfaces/muon_fitting_tab.rst (hunk 2 above)
Comment (Harrietbrown) [reply docs/source/interfaces/muon_fitting_tab.rst]:
```suggestion
The fitting browser here is similar to that used in the :ref:`Multi-dataset Fitting Interface <Multi-dataset-Fitting-Interface>`.

```

**Review by AnthonyLim23 — APPROVED** [review]

**Pull request merged**